inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -1342,7 +1342,7 @@ class ScaleFactorCalculator {
     template<class T>
     static int GetMandatoryWeightsBytesSize(T ptr) {
         auto info = LayerInfo(ptr);
-        if (info.isConvolution() || info.isScaleShift()) {
+        if (info.isConvolution() || info.isScaleShift() || info.isConcatAlignFilter()) {
             return GetOptionalWeightsBytesSize();
         }
 
```

Comment:
can you add some test for this problem? for example some simple network with FakeQuantize which also contains some weightable layer with not-quantized weights (or this layer is added by the plugin)